.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
```

Comment:
It may be better to run on pull requests as well to validate them

Reply:
Afaik, this already includes pull requests

Reply:
I'm no expert on GitHub actions, but the documentation makes it sound like `pull_request` is a different trigger: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
I tested it with this PR (https://github.com/28Smiles/qcvis/pull/1), but it is possible, that the `pull_request` will use the trunks workflow and the `push` trigger the pr's workflow.

Reply:
Ok, I guess we can leave this "let's wait and see" and adjust if necessary